Resources/assets_src/src/js/page.js

```diff
@@ -492,4 +492,13 @@ Sonata.Page = {
 
         }
     }
-}
\ No newline at end of file
+};
+
+
+// auto-initialize plugin
+$(function () {
```

Comment:
Yes, it does as expected. It is called when DOM is ready. It is the shorthand version for it (https://learn.jquery.com/using-jquery-core/document-ready/)